### ISSUE TITLE
Bat Rider Guards

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5685]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There are missing spawns of Bat Rider Guards in thrallmar, I only found 1 that keeps patrolling in the air and it also never attacks enemy players that come there to gank.
**Expected behaviour**: Tell us what should happen instead
There should be way more guards patrolling around and also attacking enemy players whenever they come attempting to knock them off their mounts according to wowwiki
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
https://wowwiki.fandom.com/wiki/Bat_Rider_Guard
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
here i took 3 screenshots from dala wow and 3 screenshots from a know wotlk srv i wont name, to make a comparison
dala wow:
http://prntscr.com/mgw8a5
http://prntscr.com/mgw8i8
http://prntscr.com/mgw8q3
other wotlk srv:
http://prntscr.com/mgw93a
http://prntscr.com/mgw9b3
http://prntscr.com/mgw9iu

**Comment by arestavo** [comment]
I've been to many an allied town in outland/northrend and I've never had a guard's mark put on me or been attacked by a flying guard.

Pretty sure the scripting is 100% broken, unless that was how it was supposed to be in wrath.
